src/devices/Mcp25xxx/Register/AcceptanceFilter/RxFxEid8.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Mcp25xxx.Register.AcceptanceFilter
+{
+    /// <summary>
+    /// Filter Extended Identifier High Register.
+    /// </summary>
+    public class RxFxEid8 : IRegister
+    {
+        /// <summary>
+        /// Initializes a new instance of the RxFxEid8 class.
+        /// </summary>
+        /// <param name="rxFilterNumber">Receive Filter Number.</param>
+        /// <param name="eid">Extended Identifier bits.</param>
+        public RxFxEid8(RxFilterNumber rxFilterNumber, byte eid)
+        {
+            RxFilterNumber = rxFilterNumber;
+            Eid = eid;
+        }
+
+        /// <summary>
+        /// Receive Filter Number.
+        /// </summary>
+        public RxFilterNumber RxFilterNumber { get; }
+
+        /// <summary>
+        /// Extended Identifier bits.
+        /// </summary>
+        public byte Eid { get; }
+
+        private Address GetAddress()
+        {
+            switch (RxFilterNumber)
+            {
+                case RxFilterNumber.Zero:
```

Comment:
If you use int as filter number then you can just have a static array somewhere:
```
Address[] s_filterAddresses = new Address[] { Address.RxF0Eid8, Address.RxF1Eid8, ... };
```

and use indexer here

Reply:
I can see this being useful, but each Register usually has 2 methods for the conversion and they use different register values.  I was going to try a helper, but I think the way it is now is clean enough.  We can circle back on this one later if needed.  I did update to use the int instead of enum from above feedback.